Swipeable.js

```diff
@@ -8,6 +8,10 @@ var Swipeable = React.createClass({
     onSwipingRight: React.PropTypes.func,
     onSwipingDown: React.PropTypes.func,
     onSwipingLeft: React.PropTypes.func,
+    onSwipedLeft: React.PropTypes.func,
```

Comment:
Last request. It may seem silly but I'd like to keep ordering like in CSS properties.

Up, Right, Down, Left. Just these and the README needs to change.